configure.librdkafka

```diff
@@ -42,7 +42,10 @@ function checks {
     # optional libs
     mkl_lib_check "zlib" "WITH_ZLIB" disable CC "-lz"
     mkl_lib_check "libcrypto" "" disable CC "-lcrypto"
-    mkl_lib_check "liblz4" "WITH_LZ4" disable CC "-llz4"
+
+    if mkl_compile_check "llz4" "WITH_LZ4" disable CC "#include <lz4frame.h>"; then
```

Comment:
I wonder if maybe mkl_lib_check should be changed to compile the snippet (if any) even if pkg-config succeeds.